### ISSUE TITLE
Suspected errors in PIRS-701

### ISSUE THREAD

**Issue by ussatjam** [nrc-cnrc/EGSnrc#967]
I recently re-read the pair production and bremsstrahlung sections of PIRS-701 and found several apparent errors. Some are just typos, but some look to me like mathematical errors in the equations. The complete list is contained in the attached MS Word document. Thank you.
[Corrections_to_PIRS-701.docx](https://github.com/nrc-cnrc/EGSnrc/files/10758195/Corrections_to_PIRS-701.docx)

**Comment by ftessier** [comment]
Thanks @ussatjam for the exhaustive review! 🏅 We will certainly update the manual accordingly.

**Comment by ftessier** [comment]
I converted the `.docx` to markdown so that it shows up here.

**Comment by ussatjam** [comment]
_The following text is from a follow-up email sent to Frédéric Tessier on Friday, 17 February. Frédéric asked me to post it here:_

Thanks for your note. I’ll follow along with any discussion in GitHub, but I suppose that ultimately someone on your team will need to determine if the mathematical errors that I think I found require correction or not. I’d be happy to have a one-on-one correspondence with anyone on your team who is investigating. And what I would really appreciate is if someone there could let me know about the resolution of the few really substantive issues, specifically:
 
1.	Whether the expression for ZV in equation (2.1.10) needs to be corrected as proposed.
2.	Whether the expression for δC in equation (2.1.10) is correct as written, or whether the arithmetic mean should be replaced with a geometric mean, as discussed in the Appendix of the document I sent.
3.	Whether equation (2.1.13) needs to be corrected as proposed, replacing A(δ) with A(δ)/Amax.
4.	Whether equation (4.2.7) needs to be corrected as proposed, replacing 4/3 with 16/3.
5.	Whether equation (4.2.13) needs to be corrected as proposed, moving the “2” from the denominator to the numerator.
6.	Whether kc and AP are the same or different, as discussed in the note concerning page 68 of the user manual.
7.	Whether the Coulomb correction is turned on for energies > 50 MeV or for energies ≥ 50 MeV. (A user would need to know this if running a benchmark for the nice round number of 50 MeV.)
 
I’m pretty confident about most of the proposed corrections, but if I made a mistake somewhere, I would be most grateful if someone would reach out and let me know. Thanks!
 
...
 
P.S. In a few cases, one can’t rule out the possibility of a bug in EGSnrc as well as in the documentation. For example, if 4/3 should really be in 16/3 in equation (4.2.7), then the 4/3 might be in source code as well.

**Comment by eric11210** [comment]
@ussatjam Great work!

@ftessier I would like to point out here that while we're at it that the sampling of the bremsstrahlung photon angle in the source code is not as it is described in section 2.4.2.iii, as one can see [here starting on line 465](https://github.com/nrc-cnrc/EGSnrc/blob/3cc786bb6e1ce9948e3e63285eeb09ed7c2f71b4/HEN_HOUSE/src/egsnrc.mortran#L465). That implementation seems to have been done originally, but it has been commented out and replaced with one that "is way more efficient". The documentation should be updated to reflect that. It appears in any case (at first glance) that the range of cos\theta is -1 to 1, as expected.

I also agree with replaced 4/3 with 16/3 in equation 4.2.7, for my part.

**Comment by ftessier** [comment]
Thanks @eric11210, I will indeed look into your brems derivation and correction, as previously discussed. Thank you for confirming the `4/3` → `16/3` correction.

**Comment by eric11210** [comment]
This is actually different from any brems corrections I had pointed out before. This is purely an error in the documentation, whereas the two other potential errors were actually errors in the code itself.

**Comment by ftessier** [comment]
Yes, sorry I did not make that clear. What I meant is I will _also_ look into the brems correction #855, i.e., I have not (yet) forgotten that! 😄
